app/src/main/kotlin/com/wire/android/ui/userprofile/self/SelfUserProfileScreen.kt

```diff
@@ -115,12 +115,15 @@ private fun SelfUserProfileContent(
         }
     }
     val scrollState = rememberScrollState()
+    val logoutWipeDataDialogState = rememberVisibilityState<LogoutWipeDataDialogState>()
```

Comment:
What do you think of renaming the dialog to LogoutOptionsDialog? :D and here logoutOptionsDailogState?

Reply:
Nice idea, I changed the name 👍